src/terrain/terrain.hpp

```diff
@@ -97,34 +98,17 @@ getter_low(Tile* t) {
  * path-finding and its own generation.
  *
  */
-class Terrain : public voxel_utility::VoxelBase {
+class Terrain : voxel_utility::VoxelBase{
     friend class AdjacentIterator;
+    friend class Chuck;
```

Comment:
C H U N K

Reply:
why did this work?

Reply:
Because you never accessed Chunk's private parts. 

Delete the line you dont want friends in CPP